lib/plugins/aws/provider/awsProvider.js

```diff
@@ -146,11 +146,21 @@ class AwsProvider {
         f()
           .then(resolve)
           .catch((e) => {
-            if (e.statusCode === 429) {
+            const err = e;
+            if (err.statusCode === 429) {
               that.serverless.cli.log("'Too many requests' received, sleeping 5 seconds");
               setTimeout(doCall, 5000);
             } else {
-              reject(e);
+              if (err.message === 'The security token included in the request is invalid') {
+                const errorMessage = [
+                  'The AWS security token included in the request is invalid.',
+                  ' Please check your ~./aws/credentials file and verify your keys are correct.',
+                  ' More information on setting up credentials',
+                  ` can be found here: ${chalk.green('https://git.io/vXsdd')}.`,
```

Comment:
Could we update this link so that it points to https://serverless.com/framework/docs/providers/aws/guide/credentials/ instead of the GitHub repo page?